src/clndr.js

```diff
@@ -638,6 +638,15 @@
       eventType = 'touchstart';
     }
 
+    // Make sure we don't already have events
+    $container.off( eventType +'.clndr', '.'+this.options.targets.day )
+    .off( eventType +'.clndr', '.'+this.options.targets.empty )
+    .off( eventType +'.clndr', '.'+self.options.targets.previousButton )
+    .off( eventType +'.clndr', '.'+self.options.targets.nextButton )
+    .off( eventType +'.clndr', '.'+self.options.targets.todayButton )
+    .off( eventType +'.clndr', '.'+self.options.targets.nextYearButton )
+    .off( eventType +'.clndr', '.'+self.options.targets.previousYearButton );
+
```

Comment:
@lukefrake Should all `self` references instead use `this` here? This is mixing both but you don't need to use `self` in this context.

Also, can you reformat this to have the `.off` indented? Lines 694 to 699 have a syntax example. Like:
```
$container
  .off( eventType +'.clndr', '.'+this.options.targets.day )
  .off( eventType +'.clndr', '.'+this.options.targets.empty )
  .off( eventType +'.clndr', '.'+this.options.targets.previousButton )
  ...
```